configure

```diff
@@ -3510,6 +3516,32 @@ if test -n "$_host"; then
 			append_var LDFLAGS "-static-libgcc"
 			append_var LIBS "-laudio"
 			;;
+		miyoo)
```

Comment:
Couldn't this be merged in opendingux-* branch?

Reply:
It could but it results in kinda spaghetti code because opendingux is made of 2 parts: 1) flags and it assumes mips there. 2) supported features. It's different on miyoo. Miyoo is kinda opendingux but not quiet